controllers/operands/kubevirt_test.go

```diff
@@ -2797,6 +2797,47 @@ Version: 1.2.3`)
 			})
 		})
 
+		Context("Virtual machine options", func() {
+			It("should set disableFreePageReporting by default", func() {
+				kv, err := NewKubeVirt(hco)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(kv.Spec.Configuration).To(Not(BeNil()))
+				Expect(kv.Spec.Configuration.VirtualMachineOptions).To(Not(BeNil()))
+				Expect(kv.Spec.Configuration.VirtualMachineOptions.DisableFreePageReporting).To(Not(BeNil()))
```

Comment:
`DisableFreePageReporting` is not a pointer. We can't use this assertion.
```suggestion
				Expect(kv.Spec.Configuration.VirtualMachineOptions.DisableFreePageReporting).To(BeTrue())
```

Reply:
Well, actually, kubevirt `DisableFreePageReporting` is a pointer https://github.com/kubevirt/hyperconverged-cluster-operator/blob/c3cea1fba78f9fcd4bffaf1723f395ee69725996/vendor/kubevirt.io/api/core/v1/types.go#L2451
So I think that we are ok here. WDYT?

Reply:
Sorry - misread. It's on KV...

Anyway, you can also check its value.

```suggestion
				Expect(kv.Spec.Configuration.VirtualMachineOptions.DisableFreePageReporting).To(HaveValue(BeTrue()))
```

Reply:
`DisableFreePageReporting` is a pointer to an empty struct: the existence of the struct means that free page reporting is disabled, the non existence means free page reporting is enabled. I think that there is nothing more to check here. Do you agree? Thanks